homeassistant/components/sabnzbd/__init__.py

```diff
@@ -254,7 +78,6 @@ async def async_service_handler(service: ServiceCall) -> None:
 
     async def async_update_sabnzbd(now):
```

Comment:
Why does this service exist? We already have a `homeassistant.update_entity` service

Reply:
this is in order to update stats from sabnzbd every UPDATE_INTERVAL (timedelta(seconds=30))

Reply:
Oops, thought this was a service call but it's for time. This is ok.